sections/browsers.include

```diff
@@ -59,11 +59,31 @@
   <a>creator browsing context</a>. Otherwise, the <a>browsing context</a> has no
   <a>creator browsing context</a>.
 
-  If a <a>browsing context</a> <var>A</var> has a <a>creator browsing context</a>, then the
-  <dfn>creator origin</dfn>, <dfn>creator URL</dfn>, and <dfn>creator base URL</dfn> are the
-  <a for="concept">origin</a>, <a lt="document url">URL</a>, and <a lt='document base URL'>base URL</a>, respectively, of
-  the {{Document}} that was the <a>active document</a> of that <a>creator browsing context</a> at
-  the time <var>A</var> was created.
+  <p>If a <span>browsing context</span> <var>context</var> has a <span>creator browsing
+  context</span> <var>creator</var>, it also has the following properties. In what follows, let
+  <var>creator document</var> be <var>creator</var>'s <span>active document</span> at the time
+  <var>context</var> is created:</p>
+
+  <dl>
+   <dt><dfn>creator origin</dfn></dt>
+   <dd><var>creator document</var>'s <span>origin</span></dd>
+
+   <dt><dfn>creator URL</dfn></dt>
+   <dd><var>creator document</var>'s <span data-x="concept-document-url">URL</span></dd>
+
+   <dt><dfn>creator base URL</dfn></dt>
+   <dd><var>creator document</var>'s <span data-x="document base URL">base URL</span></dd>
+
+   <dt><dfn>creator referrer policy</dfn></dt>
+   <dd><var>creator document</var>'s <span data-x="concept-document-referrer-policy">referrer
+   policy</span></dd>
+
+   <dt><dfn>creator context security</dfn></dt>
+   <dd>The result of executing <span>Is environment settings object a secure context?</span> on
+   <var>creator document</var>'s <span>relevant settings object</span></dd>
+  </dl>
+
+  <div w-nodev>
```

Comment:
The 'creator Document' was removed, and this patch caches the security status of that Document. Needed as dependency for Secure Contexts to get past CR: https://www.w3.org/TR/secure-contexts/